addons/l10n_mz/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': "Mozambique - Accounting",
+    'description': """
+		Mozambican Accounting localization
+    """,
+    'version': '1.0',
+    'category': 'Accounting/Localizations/Account Charts',
+
```

Comment:
No need need for blank lines like this in the manifest. Idem for the one a little lower.